Dockerize Applet for testing convenience

jdk24 opened the pull request.

Commit:
Create Dockerfile

chrismayer commented on raster_messenger/Dockerfile:
```diff
@@ -0,0 +1,24 @@
+ENV TEST_REGION="TestRegion"
+ENV TEST_POLLUTANT="TestSchad"
+
+RUN apt-get update && apt-get -y install -qq cron
```
Necessary to install `cron`?

jdk24 replied:
No, just the usual: 

&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;Good
&ensp;&ensp;&ensp;/&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;\\
 Fast &ensp; &ensp; \__me\__ &ensp;&ensp;  Cheap

Will commit after all comments resolved.

chrismayer reviewed (approved):
LGTM, just one minor remark

JakobMiksch reviewed (approved):
Looks alright. One minor comment inline.

JakobMiksch commented on raster_messenger/Dockerfile:
```diff
@@ -0,0 +1,24 @@
+# build the executable JAR file
+RUN mvn clean package -DskipTests
+
+CMD umchannel=$UM_CHANNEL umserver=$UM_SERVER testRegion=$TEST_REGION testPollutant=$TEST_POLLUTANT java -jar target/test-messenger-jar-with-dependencies.jar
```
I find it confusing that the variable names are not uppercase.

jdk24 replied:
As in 
```suggestion
CMD UMCHANNEL=$UM_CHANNEL ...
```
 for Java args?

JakobMiksch replied:
you probably would need to change that in the Java program as well.

I just remember that environment variables are always uppercase. 

But we can just keep it if changing has to many side effects. This was just an observation and is not blocking  ...

Commit:
Reflect review change requests

- Delete cron installation from Dockerfile
- Change Java CLI params to uppercase
- Add time offset param: Use OFFSETHRS=x(int) to simulate timestamp x-hours in the future in order send message with other than just current timestamp for testing

jdk24 commented:
Changes applied. 
Mind the run params now in uppercase, additional param OFFSETHRS.
Use OFFSETHRS=1 to send message for one hour in the future, so to be able to send more than current timestamp for testing
